src/chainrules.jl

```diff
@@ -38,9 +44,15 @@ end
 function _Ω_∂_xlog1py(x::Real, y::Real)
     log1py = log1p(y)
     z = x * log1py
-    Ω = iszero(x) && !isnan(y) ? zero(z) : z
+    w = x / (1 + y)
+    if iszero(x) && !isnan(y)
+        Ω = zero(z)
+        ∂y = zero(z)
```

Comment:
I think this should be
```suggestion
        ∂y = zero(w)
```
?

Reply:
Yes, will fix in next commit